src/imex.rs

```diff
@@ -115,13 +113,6 @@ pub async fn imex(
     res
 }
 
-async fn cleanup_aborted_imex(context: &Context, what: ImexMode) {
-    if what == ImexMode::ImportBackup {
-        dc_delete_file(context, context.get_dbfile()).await;
```

Comment:
This code was never tested (as evidenced by its removal not triggering any tests) and cannot work on Windows: it is not allowed to remove open files there. As for Linux, removing current database works, but the database remains open and gets actually deleted when you close the context.

This cleanup code probably worked correctly when importing was done by closing your database and replacing with backup database, but now it's definitely wrong.